src/main/python/comment_parser.py

```diff
@@ -0,0 +1,123 @@
+import os
+import pathlib
+import re
+import tokenize
+import csv
+from collections import namedtuple
+
+MAIN_RESOURCE_DIR = '../resources'
+JAVA_FILE_EXTENSION = '.java'
+PYTHON_FILE_EXTENSION = '.py'
+
+# TODO: add C# here
+extension_mapping = {
+    'java': JAVA_FILE_EXTENSION,
+    'python': PYTHON_FILE_EXTENSION
+}
+
+Comment = namedtuple('Comment', 'comment_text start_line end_line')
+
+
+def get_directories():
+    return os.listdir(MAIN_RESOURCE_DIR)
+
+
+def create_directory_for_search_phrase(directory_name):
```

Comment:
There is a function that does the same thing in another file. Can we create a new file and a function called create_directory and then import it in these two files and call wherever needed?

Reply:
sure